contrib/ontology_annotations.csv

```diff
@@ -0,0 +1,73 @@
+observationVariableName,traitDescription,scaleName,methodName,methodName,ontologyDbId,URL,traitName,comments

```

Comment:
the goal of this table is to link variable names in the database to ontology terms
* here, ontologyDbId should be renamed ontologyReference; 
* create a new field ontologyDbId that links to the plant trait ontology record in ontologies.json. this will be the same in every row
* Add observationVariableDbId to this table
* later, when constructing the observationvariable response, use this table to generate the ontologyreference